var/spack/repos/builtin/packages/udunits/package.py

```diff
@@ -10,7 +10,7 @@ class Udunits(AutotoolsPackage):
     """Automated units conversion"""
 
     homepage = "https://www.unidata.ucar.edu/software/udunits"
-    url = "https://artifacts.unidata.ucar.edu/repository/downloads-udunits/udunits-2.2.28.tar.gz"
+    url = "https://artifacts.unidata.ucar.edu/repository/downloads-udunits/2.2.28/udunits-2.2.28.tar.gz"
 
     version("2.2.28", sha256="590baec83161a3fd62c00efa66f6113cec8a7c461e3f61a5182167e0cc5d579e")
```

Comment:
Confirmed this sha256 still applies with the new URL and note all of the older appear to no longer be available.